.gitignore

```diff
@@ -106,6 +106,7 @@ venv.bak/
 
 #IDEA
 .idea
+.vscode/
```

Comment:
what generates this?

Reply:
vscode. Its an IDE/Editor

https://code.visualstudio.com/